fastify.js

```diff
@@ -281,10 +281,14 @@ function build (options) {
         server.once('error', errEventHandler)
       })
       var listen = new Promise((resolve, reject) => {
-        server.listen(port, address, backlog, () => {
-          server.removeListener('error', errEventHandler)
-          logServerAddress(server.address(), options.https)
-          resolve()
+        server.listen(port, address, backlog, (err) => {
+          if (err) {
```

Comment:
There is no need to the listen for the error here, see https://github.com/fastify/fastify/pull/965.